frontend/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Data Protection</title>
+</head>
+<body>
+	<script type="text/javascript">
+		var	object_id = Date.now();
+		alert(object_id);
+	</script>
+
+	<section>
+		<div>
+			<form>
+				<!--File Upload-->
+				<div>
+					<div>
+						<label>Upload File</label>
+					</div>
+					<div>
+						<label>File:</label>
+					</div>
+					<div>
+						<input type="file" id="myfile" name="myfile">
+					</div>
+				</div>
+
+				<!--Policy-->
+				<div>
+					<div>
+						<label>Raid:</label>
+					</div>
+					<div>
+						<select>
+							<option>RAID0</option>
+							<option>RAID1</option>
+						</select>
+					</div>
+				</div>
+				<div>
+					<input type="submit" value="Upload File">
+				</div>
```

Comment:
I suggest only a submit button in a single form. We have four main functions so I think we can divide it into four form labels